go/logic/inspect.go

```diff
@@ -278,6 +275,41 @@ func (this *Inspector) validateGrants() error {
 	return this.migrationContext.Log.Errorf("User has insufficient privileges for migration. Needed: SUPER|REPLICATION CLIENT, REPLICATION SLAVE and ALL on %s.*", sql.EscapeName(this.migrationContext.DatabaseName))
 }
 
+// grantContainsAll verifies the passed grant contain all the passed strings and
+// that the grant match the DB
+func (this *Inspector) grantContainsAll(grant string, substrings ...string) bool {
```

Comment:
@Gonlo2 is it possible to add a unit test that confirms the logic in `.grantContainsAll()`?

Reply:
@timvaillancourt Any update about this?